common/esxi_get_guest_config_options.yml

```diff
@@ -0,0 +1,312 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get default VM config options for a specific guest id
+# Parameter
+#   guest_id: guest id
+#   esxi_hardware_version: ESXi hardware version
+# Return:
+#   guest_config_options: The guest ID config options for a hardware version
+
+- block:
```

Comment:
I think here esxi_hardware_version can be a required parameter must to be set when calling this task, and no need to call esxi_get_hardware_versions.yml one more time.